main.nf

```diff
@@ -91,24 +103,50 @@ process fusioninspector {
 */
 // Requires raw untrimmed files. FastQ files should not be merged!
 process fusioncatcher {
+    tag "$name"
+
+    publishDir "${params.outdir}/FusionCatcher",  mode: 'copy'    
 
     input:
-    set val (name), file(read1), file(read2) from fusioncatcher_reads
+    set val (name), file(reads) from fusioncatcher_reads
 
     output:
-    file '*.txt' into fusioncatcher
+    file '*.{txt,log,zip}' into fusioncatcher
 
     when: params.fusioncatcher
 
     script:
     """
-    fusioncatcher \
-        -d $fusioncatcher_data_dir \
-        -i ${read1},${read2} \
-        --threads ${task.cpus} \
-        --$SENSITIVITY \
-        -o ${SAMPLE_NAME}/
+    fusioncatcher \\
+        -d ${params.fusioncatcher_data_dir} \\
+        -i ${reads[0]},${reads[1]} \\
+        --threads ${task.cpus} \\
+        --${params.sensitivity} \\
+        -o . \\
+        ${params.fc_extra_options}
+    for f in *.{txt,log,zip}
+    do
+    mv \$f $name\$f
+    done
+    """
+}
+
+process fusion_genes_compare {
+    
+    publishDir "${params.outdir}/Comparative_shortlist",  mode: 'copy'
+    
+    input:
+    file ('*star-fusion.fusion_candidates.final.abridged') from fusion_candidates_list.collect() 
+    file ('*summary_candidate_fusions.txt') from fusioncatcher.collect()
+    
+    output:
+    file '*fusion_comparison.txt'  
+    
```

Comment:
I think this needs to be conditional to only run when we have both tools in use?

```groovy
when: params.star && params.inspector
```